assets/src/containers/ResourcesAccessed.js

```diff
@@ -220,14 +221,17 @@ function ResourcesAccessed (props) {
       return (<AlertBanner>Resource data for your selections is not available.</AlertBanner>)
     } else {
       return (
+        <>
+        <ResourceChartNarrative
+            data={resourceData}
+            weekRange={weekRange}
+            gradeSelection={resourceGradeFilter}
+            resourceType={resourceTypeFilter} />
         <ResourceAccessChart
           data={resourceData}
-          weekRange={weekRange}
-          gradeSelection={resourceGradeFilter}
-          resourceType={resourceTypeFilter}
           aspectRatio={0.3}
-          minHeight={350}
-        />
+          minHeight={350} />
+        </>
```

Comment:
Nit-pick: Check this using `standard` linting, seeing some issues with my VSCode setup.

Reply:
I am configured based on Standard. So Now We need to verify why it is different.

Reply:
Happy to meet with you if you want.